pkg/adaptors/authenticateMiddleware.ts

```diff
@@ -13,28 +14,45 @@ export type AuthMiddlewareVariable = {
    * @description authorization token (JWT, ES256)
    */
   token: string;
+  /*
+   * @description account id, or none if not authorized
+   */
+  accountID: Option.Option<string>;
   /*
    * @description account name, or none if not authorized
    */
   accountName: Option.Option<string>;
 };
 
+type tokenPayload = { sub: string; accountName: string };
+const tokenPayloadSchema = z.object({
+  sub: z.string(),
+  accountName: z.string(),
+});
+
 export class AuthenticateMiddlewareService {
   private readonly authTokenService: AuthenticationTokenService;
 
   constructor(authTokenService: AuthenticationTokenService) {
     this.authTokenService = authTokenService;
   }
 
-  private parseToken(token: string): Option.Option<string> {
+  private parseToken(token: string): Option.Option<tokenPayload> {
```

Comment:
```suggestion
  private parseToken(token: string): Option.Option<z.infer<typeof tokenPayloadSchema>> {
```
`tokenPayload` を手動で定義する必要は無いかと